apps/meteor/app/api/server/helpers/getLoggedInUser.ts

```diff
@@ -1,28 +1,16 @@
 import { Accounts } from 'meteor/accounts-base';
-import { Users as UsersRaw } from '@rocket.chat/models';
+import { Users } from '@rocket.chat/models';
 import type { IUser } from '@rocket.chat/core-typings';
 
-import { Users } from '../../../models/server';
-import { API } from '../api';
-
 export async function getLoggedInUser(token: string, userId: string): Promise<Pick<IUser, '_id' | 'username'> | null> {
 	if (!token || !userId) {
 		return null;
 	}
-	return UsersRaw.findOne(
+	return Users.findOne(
 		{
 			'_id': userId,
 			'services.resume.loginTokens.hashedToken': Accounts._hashLoginToken(token),
```

Comment:
I think we have a query for this on the model 🤔 

But, btw, you can also use the same _hashLoginToken from account-utils package